x/payment/keeper/grpc_query_params.go

```diff
@@ -0,0 +1,19 @@
+package keeper
+
+import (
+	"context"
+
+	"github.com/celestiaorg/celestia-app/x/payment/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func (k Keeper) Params(c context.Context, req *types.QueryParamsRequest) (*types.QueryParamsResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+	ctx := sdk.UnwrapSDKContext(c)
+
+	return &types.QueryParamsResponse{Params: k.GetParams(ctx)}, nil
+}
```

Comment:
[doesn't have to be done in this PR][new good first issue]
Its might be a bit extravagant to have this level of test coverage, but we can even include a test for querying these params in cli testutil

Reply:
could also have a test to update these params, but again, that's kinda extravagant.

Reply:
[Blocking]

> but we can even include a test for querying these params in cli testutil

:+1: 

I guess it is really important to test it either using a test of confirming that the params are queried correctly from the CLI.
I had some fun errors when querying params before for the QGB, even if the implementation looked right. Thus, I would prefer if we can get a confirmation on this.

Reply:
Done, please re-review